docs/source/markdown/podman-ps.1.md

```diff
@@ -55,7 +55,8 @@ Valid placeholders for the Go template are listed below:
 | **Placeholder** | **Description**                                  |
 | --------------- | ------------------------------------------------ |
 | .ID             | Container ID                                     |
-| .Image          | Image ID/Name                                    |
+| .Image          | Image Name/ID                                    |
+| .ImageD         | Image ID                                         |
```

Comment:
Did you mean 'ImageD' instead of 'ImageID'?

Reply:
Good catch! fixed now :)